ClientPackages/dirac-make.py

```diff
@@ -30,10 +30,8 @@
 
 ch.setPackageVersions( versions )
 
-for package in versions:
-  packageToInstall = "%s>=%s" % ( package, versions[ package ] )
-  if not ch.easyInstall( packageToInstall ):
-    logging.error( "Could not deploy %s with easy_install", package )
-    if not ch.pip( packageToInstall ):
-      logging.error( "Could not deploy %s with pip", package )
-      sys.exit( 1 )
+for package, version in versions.iteritems():
+  packageToInstall = "%s=%s" % (package, version)
```

Comment:
I think this is a bug. Should be:
packageToInstall = "%s==%s" % (package, version)

Reply:
right. Fixing it

Reply:
done.